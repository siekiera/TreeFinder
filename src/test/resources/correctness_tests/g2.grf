8
12
0.0 0.0
1.0 1.0
1.0 0.0
1.0 -1.0
2.0 1.0
2.0 0.0
2.0 -1.0
3.0 0.0
1 2 1
1 3 2
1 4 3
2 3 4
2 5 1
3 4 5
3 6 1
4 6 5
5 6 1
6 7 5
6 8 2
7 8 6