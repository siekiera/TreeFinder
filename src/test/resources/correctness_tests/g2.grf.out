8
7
0.0 0.0
1.0 1.0
1.0 0.0
1.0 -1.0
2.0 1.0
2.0 0.0
2.0 -1.0
3.0 0.0
1 2 1
1 4 3
2 5 1
3 6 1
5 6 1
6 7 5
6 8 2